scanner/e2etests/helmchart/templates/scanner-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.app.scanner.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.app.scanner.name }}
+spec:
+  replicas: 1
+  minReadySeconds: 15
+  selector:
+    matchLabels:
+      app: {{ .Values.app.scanner.name }}
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      labels:
+        app: {{ .Values.app.scanner.name }}
+    spec:
+      securityContext:
+        fsGroup: 65534  # Set the group ownership to the `nobody` group, see dockerfile.
+      containers:
+      - name: scanner
+        image: "{{ ( printf "%s/%s:%s" .Values.image.repository .Values.app.scanner.name .Values.image.tag ) }}"
+        imagePullPolicy: IfNotPresent
+        args:
+          - -conf
+          - /etc/stackrox.d/scanner/config.yaml
+        resources:
+          limits:
+            cpu: 2
+            memory: 4Gi
+          requests:
+            cpu: 200m
+            memory: 200Mi
+        volumeMounts:
```

Comment:
Have there not been issues with running ClairCore in "online" mode? Is this not needed?

```
        - name: etc-ssl-volume
          mountPath: /etc/ssl
        - name: etc-pki-volume
          mountPath: /etc/pki/ca-trust
```

See https://github.com/stackrox/stackrox/pull/7706/files#diff-5edced4c8d492266b83ec518684955a5b56186cbb5771e70c0db1bc1400f7997 for more info

Reply:
Added anyway: 9793cde4ea